src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -230,7 +230,7 @@ public function addXmlContent($content, $charset = 'UTF-8')
         $dom->validateOnParse = true;
 
         if ('' !== trim($content)) {
-            @$dom->loadXML($content, LIBXML_NONET);
+            @$dom->loadXML($content, LIBXML_NONET | LIBXML_PARSEHUGE);
```

Comment:
Anyway I can find only one dist with libxml2 < 2.7.3 - CentOS 5
Even on Debian 6 it 2.7.8